PROCESSORS.md

```diff
@@ -208,6 +209,39 @@ In the list below, the names of required properties appear in bold. Any other pr
 |success|FlowFiles that are sent successfully to the destination are transferred to this relationship|
 
 
+## DeleteS3Object
+
+### Description
+
+Deletes FlowFiles on an Amazon S3 Bucket. If attempting to delete a file that does not exist, FlowFile is routed to success.
```

Comment:
This sounds like "if something does not exist it is rerouted to success".

Reply:
I see this is verbatim from NiFi, but I still don't understand what is going on. If a flowfile does not exist, this processor yields.

Reply:
We started discussing this with @arpadboda and @adebreceni in a comment above, where @adebreceni noted that it can be logical if we interpret it the following way: "if you define "success" as the post-condition "the object is not in the S3 bucket", it makes sense to transfer to "success"". So if the goal is to have the object non-existent in that bucket we succeed by doing nothing.